src/plugins/console/public/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -228,6 +228,11 @@ function EditorUI({ initialTextValue, dataSourceId }: EditorProps) {
     });
   }, [sendCurrentRequestToOpenSearch, openDocumentation]);
 
+  const tooltipDefaultMessage =
+    dataSourceId === undefined ? 'Select a Data Source' : `Click to send request`;
```

Comment:
It should be `Select a data source`, not `Select a Data Source`. Can you please update?